ADALiOS/ADALiOS/ADLogger+Internal.h

```diff
@@ -0,0 +1,93 @@
+#define AD_LOG_VERBOSE(_message, _info)         AD_LOG(ADAL_LOG_LEVEL_VERBOSE, _message, AD_ERROR_SUCCEEDED, _info)
+
+#define AD_LOG_ERROR_F(_msg, _code, _fmt, ...)        AD_LOG_F(ADAL_LOG_LEVEL_ERROR, _msg, _code, _fmt, ##__VA_ARGS__)
+#define AD_LOG_WARN_F(_msg, _fmt, ...)                AD_LOG_F(ADAL_LOG_LEVEL_WARN, _msg, AD_ERROR_SUCCEEDED, _fmt, ##__VA_ARGS__)
+#define AD_LOG_INFO_F(_msg, _fmt, ...)                AD_LOG_F(ADAL_LOG_LEVEL_INFO, _msg, AD_ERROR_SUCCEEDED, _fmt, ##__VA_ARGS__)
+#define AD_LOG_VERBOSE_F(_msg, _fmt, ...)             AD_LOG_F(ADAL_LOG_LEVEL_VERBOSE, _msg, AD_ERROR_SUCCEEDED, _fmt, ##__VA_ARGS__)
+
+#ifndef DebugLog
+#ifdef DEBUG
+#   define DebugLog(fmt, ...) NSLog((@"%s[%d][%@] " fmt), __PRETTY_FUNCTION__, __LINE__, [[NSThread currentThread] isEqual:[NSThread mainThread]] ? @"main" : @"work", ##__VA_ARGS__);
+#else
+#   define DebugLog(...)
+#endif
+#endif
+
+@interface ADLogger (Internal)
+
+/*! Returns diagnostic trace data to be sent to the Auzure Active Directory servers. */
++ (NSDictionary*)adalId;
+
+/*! Calculates a hash of the passed string. Useful for logging tokens, where we do not log
+ the actual contents, but still want to log something that can be correlated. */
++ (NSString*)getHash:(NSString*)input;
+
++ (NSString*)getAdalVersion;
+
++ (NSString*)getCPUInfo;
+
+/*! Returns previously set callback call or nil, if the user has not set such callback. */
++ (LogCallback)getLogCallBack;
+
+/*! Main logging function. Macros like ADAL_LOG_ERROR are provided on top for convenience
+ @param logLevel: The applicable priority of the logged message. Use AD_LOG_LEVEL_NO_LOG to disable all logging.
+ @param message: Short text defining the operation/condition.
+ @param additionalInformation: Full details. May contain parameter names, stack traces, etc. May be nil.
+ @param errorCode: if an explicit error has occurred, this code will contain its code.
+ */
++ (void)log:(ADAL_LOG_LEVEL)logLevel
+    message:(NSString*)message
+  errorCode:(NSInteger)errorCode
+       info:(NSString*)additionalInformation;
+
+/*! Convience logging fucntion. Allows the creation of additionalInformation strings using format strings. */
++ (void)log:(ADAL_LOG_LEVEL)level
+    message:(NSString*)message
+  errorCode:(NSInteger)code
+     format:(NSString*)format, ... __attribute__((format(__NSString__, 4, 5)));
+
+/*! Logs obtaining of a token. The method does not log the actual token, only its hash.
+ @param token: the token to log.
+ @param tokenType: "access token", "refresh token", "multi-resource refresh token"
+ @param expiresOn: the time when an access token will stop to be valid. Nil for refresh token types.
+ @param correlationId: In case the token was just obtained from the server, the correlation id of the call.
+ This parameter can be nil.
+ */
++ (void)logToken:(NSString*)token
+       tokenType:(NSString*)tokenType
```

Comment:
We're just printing strings here, not performing any actual logic on them. This also is previous implementation. Creating an enum would be overkill.